update_copyrights

```diff
@@ -115,15 +120,20 @@ if len(removed) > 0:
     any_check_failed = True
     print("There are "+str(len(removed))+" removed images")
 
-if len(added) > 0 or len(changed) > 0:
+if len(added) > 0 or len(changed) > 0 or len(incomplete) > 0:
     any_check_failed = True
     print("There are "+str(len(added))+" new images")
     print("There are "+str(len(changed))+" changed images")
+    print("There are "+str(len(incomplete))+" images that lack license or author information")
+    if options.output != "":
+        print("These are at the top of the output for easy editing, run the tool again after editing to sort the file")
 
 if csvfile_needs_sorting:
     any_check_failed = True
     print("The input file isn’t sorted by filename")
```

Comment:
This seems like more of a debug message than something helpful for normal use, since it's complaining about something that isn't an error.  I'd say the message added below at Line 136 is confirmation enough.

Reply:
This is one of the things that the CI should treat as an error, so I want to leave the message in.

If the script overwrote the input file in-place then I'd agree about suppressing it, but in the current `--output <other_file.csv>` workflow this message indicates that committing the current code and uploading would trigger an error when the CI tests it.